week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,28 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+//The percentage function
+function percentage(numberOfStudents, numberOfMentors) {
+  var total = numberOfStudents + numberOfMentors;
+  var percentageOfStudents = (numberOfStudents / total) * 100;
+  var percentageOfMentors = (numberOfMentors / total) * 100;
+  printPercentage(percentageOfStudents, percentageOfMentors);
```

Comment:
I really struggled finding a way to return two values on this task - I ended up returning an array -  and this solution is very good! The way in which the total is generated through adding the remaining variables is also very creative and clean.

Reply:
nice work @ahmedahashim .

Reply:
This solution is quite creative, yes, but you don't need to calculate both at the same time! One job of a developer is to look for improving code and adhering do the DRY principle (do you know what it means?). In here - you're calculating the student percentage and then the mentors - but are you sure you have to? `numberOfStudents` is just a variable - you always obtain a percentage of something in the same way! You just need the total and the number of "things". So - how can this be improved then, without using an array either?